data/json/mutations/mutations.json

```diff
@@ -6394,7 +6397,7 @@
     "visibility": 10,
     "ugliness": 8,
     "consume_time_modifier": 0.3,
-    "description": "Your teeth have grown incredibly sharp and are formed of very dense calcium material.  In addition to making you eat much faster, you can use them as a lethal natural weapon, as long as your anatomy favors it.  They also grow very fast, and you harmlessly shed them more or less at random.",
+    "description": "Your teeth have grown incredibly sharp, and are formed of very dense calcified material.  In addition to letting you eat much faster, you can use them as a lethal natural weapon, as long as your anatomy favors it.  They also grow very fast, and you harmlessly shed them more or less at random.",
```

Comment:
```suggestion
    "description": "Your teeth have grown incredibly sharp, and are formed of a very dense calcium-based material.  In addition to letting you eat much faster, you can use them as a lethal natural weapon, as long as your anatomy favors it.  They also grow very fast, and you harmlessly shed them more or less at random.",
```

Reply:
"Calcified" means "hardened by deposition of or conversion into calcium carbonate or another insoluble calcium compound." Given the nature of this as mutation, it seems more appropriate.

Reply:
It's just that teeth are already made of (denser than bone, for both enamel and dentin - see [this free article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4391782); enamel has at least twice the density of bone) calcified material.

Reply:
"Your teeth have grown incredibly sharp, and have calcified even further"?

Reply:
Yes; "hardened even further" (or maybe "gotten even harder") would also work. (Calcified further does make sense - teeth would be harder than they are with less protein and more calcium, but they'd also be more brittle... not a problem with continuous replacement teeth!)